src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -22,9 +26,112 @@ public Program(IAssemblyLoaderContainer container, IApplicationEnvironment envir
 
         public Task<int> Main(string[] args)
         {
-            Console.WriteLine("I AM APP HOST 2!");
-            Console.WriteLine("TODO: Actually do things!");
+            ApplicationHostOptions options;
+            string[] programArgs;
```

Comment:
programArgs is updated in ParseArgs but not actually used anywhere. Unless this is for some future use?